docs/appendix/troubleshooting.md

```diff
@@ -209,7 +209,7 @@ It will resemble the following:
 
       Replace `/dev/video0` in the above command with the video path you determined for your video device above, if different.
 
-      The command will return a list of pixel formats your camera supports, such as `MJPG` or `YUYV`.
+      The command will return a list of pixel formats your camera supports, such as `MJPG` (also notated as `MJPEG`) or `YUYV` (also notated as `YUY2`).
```

Comment:
Great clarification. For context, I'm pretty sure YUY2 is only used in Windows environments ([source](https://www.kernel.org/doc/html/v4.8/media/uapi/v4l/pixfmt-yuyv.html))
>  V4L2_PIX_FMT_YUYV is known in the Windows environment as YUY2.

But I don't think it matters enough to clarify here. This looks good to me and I just mention it in case you're curious.